### PR TITLE
Add link that create an issue using the bug report template

### PR DESCRIPTION
**Pull request by FirelightFlagboy**
Rework `Filing Bug` section to include a link that create an issue using the bug report template.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,11 +23,13 @@ An issue is either a **bug** (an unexpected / unwanted behavior of the software
...
+If you file a **bug issue**, use the following link to [Create a bug report](https://github.com/Scille/parsec-cloud/issues/new?template=bug-report.yml&title=[%F0%9F%90%9B+|+Bug]%3A+).
 
-1. version of Parsec (either by doing `parsec --versions`, by getting it from the GUI, or by providing the git tag if you work from the sources)
-2. error/log output (eg a traceback)
-3. steps to reproduce the issue
+It's a template that list the required info that we need about it, you will need to provide:
+
+1. The version of Parsec (either by doing `parsec --versions`, by getting it from the GUI, or by providing the git tag if you work from the sources).
+2. Error/log output (eg a traceback).
+3. Steps to reproduce the issue.
```
Comment (mmmarcos) [inline CONTRIBUTING.md:32]: I think this is redundant with the report itself. 
BTW, the instructions to obtain Parsec version should be moved to the bug report.

**Review by mmmarcos — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -23,11 +23,13 @@ An issue is either a **bug** (an unexpected / unwanted behavior of the software
 
 ### Filing Bugs
 
-If you file a **bug issue**, please provide at least the following information:
+If you file a **bug issue**, use the following link to [Create a bug report](https://github.com/Scille/parsec-cloud/issues/new?template=bug-report.yml&title=[%F0%9F%90%9B+|+Bug]%3A+).
```
[the remark this answers is not in the document]
Comment (FirelightFlagboy) [reply CONTRIBUTING.md:26]: That provide more visual hint, that also the default title set for the bug template.

So I you manually click on the button bug report you will see that the title to pre-filled with `[:bug: | Bug]`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -23,11 +23,13 @@ An issue is either a **bug** (an unexpected / unwanted behavior of the software
...
 
-1. version of Parsec (either by doing `parsec --versions`, by getting it from the GUI, or by providing the git tag if you work from the sources)
-2. error/log output (eg a traceback)
-3. steps to reproduce the issue
+It's a template that list the required info that we need about it, you will need to provide:
```
[the remark this answers is not in the document]
Comment (FirelightFlagboy) [reply CONTRIBUTING.md:28]: At that point if the list below it is redundant, this phrase also is

[the remark this answers is not in the document]
Comment (mmmarcos) [reply CONTRIBUTING.md:28]: That's why I added the "add any extra information that might help us...". I agree it's still a little redundant but at least the section is not so empty. As you prefer.

File: CONTRIBUTING.md (hunk 1 above)
Comment (FirelightFlagboy) [reply CONTRIBUTING.md:32]:
```suggestion
```

**Review by mmmarcos — APPROVED** [review]

**Pull request merged**